packages/scripts/src/application.ts

```diff
@@ -233,6 +235,28 @@ export class Application {
         };
     }
 
+    public async remote({ port: preferedPort }: IRunOptions = {}) {
+        if (!process.send) {
+            throw new Error('Cant launch remote from main process');
+        }
+
+        await this.loadEngineConfig();
+        const { socketServer, close, port } = await this.launchHttpServer(preferedPort);
+        const forkedProcess = new ForkedProcess(process);
+        createCommunication(forkedProcess, socketServer, { port, onClose: () => close() });
```

Comment:
consider renaming function `createCommunication` to avoid confusion with com feature, etc.
also: `onClose: close`. no need to create an extra fn.